website/src/pages/case-studies/nutanix-fixed-log4j-with-sourcegraph.tsx

```diff
@@ -0,0 +1,332 @@
+                        to identify the issue and start fixing it ASAP.”
+                    </p>
+                    <p>
+                        Jon explained that, at many enterprises, dependencies are unseen and forgotten. “You might check
+                        in on a dependency and find it hasn't been reviewed for four, five, six, or even ten years. But
+                        it works, so why update it?”
+                    </p>
+                    Log4j is one of many reasons why monitoring and updating is now a renewed priority. “Companies will
+                    have to be more diligent,” Jon said.
+                    <p>Sourcegraph is the best way to find and fix the next Log4j vulnerability.</p>
+                </section>
+            </ContentSection>
+
+            <AuthorBio
+                customer="Nutanix"
+                image="/case-studies/jon-kohler.png"
+                author="Jon Kohler"
+                title="Technical Director of Solution Engineering at Nutanix"
+                about="Nutanix has 20,000 customers, an annual revenue of nearly $1.394 billion, and over 6,000 employees. Organizations around the world rely on Nutanix software as a single platform to manage any app at any scale for their hybrid multicloud environments."
+            />
+
+            <div className="bg-light-gray2">
+                <ContentSection>
+                    <div className="row d-flex flex-column mx-4 mx-lg-0 py-7 align-items-lg-center align-items-left">
+                        <div className="mb-5 d-flex flex-column">
+                            <h1 className="text-center font-weight-bold">Get started with Sourcegraph</h1>
+                            <p className="text-center">
+                                Understand, fix, and automate changes across your entire codebase.
+                            </p>
+                        </div>
+                        <div className="d-flex flex-column">
+                            <a
+                                className="btn btn-primary"
+                                href="https://info.sourcegraph.com/demo-request"
+                                title="Request a Demo."
+                            >
+                                Request a demo
+                            </a>
+                            <Link to="/get-started" className="d-flex justify-content-center mt-4">
+                                <p className="font-weight-bold">Try Sourcegraph now</p>
+                            </Link>
+                        </div>
+                    </div>
+                </ContentSection>
+            </div>
+
+            <ContentSection color="white" className="py-lg-7 py-5">
+                <h1 className="pl-5 pb-5 display-3 font-weight-bold">Explore other case studies</h1>
+                <CaseStudiesList listLength={4} />
```

Comment:
One thing I am just noticing now — the cards in the design don't have borders, but these do